### ISSUE TITLE
Provide localization and internationalization

### ISSUE THREAD

**Issue by Naxos84** [NativeScript/NativeScript#7377]
**Is your feature request related to a problem? Please describe.**
the [nativescript-localize](https://www.npmjs.com/package/nativescript-localize) plugin has been deprecated and states that it has security vulnerabilities.

**Describe the solution you'd like**
I'd like to have some functionality that provides localization/internationalization for a Nativescript project (preferably Angular style)

**Describe alternatives you've considered**
I searched for other localization plugins but all other options seem to be either deprecated or unmaintained.

**Additional context**
None

Referencing #42 and the fact that [nativescript-localize plugin](https://www.npmjs.com/package/nativescript-localize) is now being deprecated and possibly has security vulnerabilities.

Are you planning to provide localization from core?

**Comment by NickIliev** [comment]
@Naxos84 try using [this library](https://www.npmjs.com/package/@localization/l10n) for your Angular project

**Comment by Naxos84** [comment]
Thx @NickIliev . Do you know wether it is possible to translate the app_name and Info.plist entries too?
If it is not possible then this is not a real option.

**Comment by NickIliev** [comment]
@Naxos84 yep, you are right - this is Angular only option but has nothing to offer for the mobile specifics.

**Comment by Naxos84** [comment]
Well then this plugin is definitely no option for me.
So my question still stands.

Are you planning to provide localization from core?

**Comment by CatchABus** [comment]
Closing as we now use the localize plugin and its community counterpart.
